backend/typescript/types.ts

```diff
@@ -0,0 +1,18 @@
+export type Role = number;
```

Comment:
Change to enum after discussing number of roles to support and deciding on role names